Projects/UOContent/Items/Weapons/BaseWeapon.cs

```diff
@@ -3423,6 +3430,91 @@ public override void OnSingleClick(Mobile from)
         from.NetState.SendDisplayEquipmentInfo(Serial, number, _crafter, false, attrs);
     }
 
+    public override void OnSingleClickPreAOS(Mobile from)
+    {
+        string prefix = null;
+        string suffix = null;
+        string slayerSuffix = SlayerNameExtensions.GetSlayerNamePreAOS(_slayer, from);
+
+        var isMagicItem = _durabilityLevel != WeaponDurabilityLevel.Regular ||
+                          _accuracyLevel != WeaponAccuracyLevel.Regular ||
+                          _damageLevel != WeaponDamageLevel.Regular ||
+                          slayerSuffix != "none";
+
+        // Construct prefix and suffix
+        var prefixBuilder = new StringBuilder();
+        var suffixBuilder = new StringBuilder();
+
+        if (isMagicItem && !_identified)
+        {
+            prefix = Localization.GetText(1038000, from.Language)?.ToLowerInvariant();
+        }
+        else
+        {
+            var qualityText = Quality != WeaponQuality.Regular
+                ? Localization.GetText(1018305 - (int)Quality, from.Language)?.ToLowerInvariant() ?? "" : "";
+
+            var durabilityText = _durabilityLevel != WeaponDurabilityLevel.Regular
+                ? Localization.GetText(1038000 + (int)_durabilityLevel, from.Language)?.ToLowerInvariant() ?? "" : "";
+
+            var accuracyText = _accuracyLevel != WeaponAccuracyLevel.Regular
+                ? Localization.GetText(1038010 + (int)_accuracyLevel, from.Language)?.ToLowerInvariant() ?? "" : "";
+
+            var damageText = _damageLevel != WeaponDamageLevel.Regular
+                ? Localization.GetText(1038015 + (int)_damageLevel, from.Language)?.ToLowerInvariant() ?? "" : "";
```

Comment:
I have code for T2A single click too, which is in the MUO discord. I didn't merge it cause the clilocs for weapon damage were deleted. I am not sure how to handle this, and it is already a problem for the existing equipment packets that Pre-AOS uses.